### ISSUE TITLE
git worktree list throws error

### ISSUE THREAD

**Issue by heytrav** [polarmutex/git-worktree.nvim#6]
I've been struggling to get this to work and after spending quite a bit of time re-reading the docs here I am not sure what it is I am doing wrong.

The `create_git_worktree` command works ok. However when I try to run the `git_worktrees()` command as explained in the README I get the following stacktrace:

```
:lua require('telescope').extensions.git_worktree.git_worktrees()                                                                                                                                          
E5108: Error executing lua ...l/share/nvim/lazy/telescope.nvim/lua/telescope/utils.lua:226: Expected Lua number
stack traceback:
        [C]: in function 'nvim_win_get_width'
        ...l/share/nvim/lazy/telescope.nvim/lua/telescope/utils.lua:226: in function 'calc_result_length'
        ...l/share/nvim/lazy/telescope.nvim/lua/telescope/utils.lua:285: in function 'transform_path'
        ...worktree.nvim/lua/telescope/_extensions/git_worktree.lua:166: in function 'parse_line'
        ...worktree.nvim/lua/telescope/_extensions/git_worktree.lua:179: in function 'git_worktrees'
        [string ":lua"]:1: in main chunk

```

I'm using Lazy for my package manager and here is my configuration to install it:

```
  {
    url = "git@github.com:polarmutex/git-worktree.nvim.git",
    opts = {
      change_directory_command = "tcd",
    },
  },

```

Any suggestions would be appreciated.

**Comment by polarmutex** [comment]
the command works for me on the main branch.  can you double check you are running the latest commit on the main branch?

**Comment by heytrav** [comment]
From the output of `:Lazy check` it looks like I am:
```
    ● git-worktree.nvim 3.43ms  start
        dir     /home/travis/.local/share/nvim/lazy/git-worktree.nvim
        url     git@github.com:polarmutex/git-worktree.nvim
        version 1.0.0
        tag     1.0.0
        branch  main
        commit  b6f0631
        readme  README.md


```

**Comment by polarmutex** [comment]
ok, I will try to reproduce it on my end

**Comment by heytrav** [comment]
> ok, I will try to reproduce it on my end
> 
> think this is the issue [ThePrimeagen/issues/112](https://github.com/ThePrimeagen/git-worktree.nvim/issues/112)

Yes that works for me as well. I switched my config to use that for and no longer get the error.  Thanks!

**Comment by jtackaberry** [comment]
@polarmutex would you consider reopening this?  @heytrav closed the issue because he applied the workaround, but I would really like to preserve `path_display = { "truncate" }` with Telescope, which is the root cause of this issue.

**Comment by polarmutex** [comment]
I will try to fix this when I get some time

**Comment by polarmutex** [comment]
think this is fixed on the version I released today (v2). I put the PR fix in my devel branch for the next version if it does not

**Comment by vivindeena** [comment]
any updates on this

**Comment by polarmutex** [comment]
> *This was generated by AI during triage.*

## Triage Notes

**What we've established so far:**

- The error occurs when Telescope is configured with `path_display = { "truncate" }`, which calls `nvim_win_get_width` on a nil window handle inside the git_worktree extension
- The reporter applied a workaround (removing the `path_display` option) but the root cause was never fixed
- Polarmutex noted this may have been addressed in the v2 release (Jul 2024)

**What we still need from you (@vivindeena or anyone hitting this):**

- Are you still seeing this error on the latest `main` with `path_display = { "truncate" }`?
- If yes, please share your full Telescope and git-worktree config, your Neovim version, and the full stack trace
